src/JaamFusion.cpp

```
@@ -127,6 +130,30 @@ int                 testTrackId = -1;
+
+SystemInfo getSystemInfo() {
+    SystemInfo info;
+    info.freeMemory = ESP.getFreeHeap();
+    info.usedMemory = ESP.getHeapSize() - info.freeMemory;
+    info.uptime = millis() / 60000;
+    info.wifiUptime = wifiConnected ? (millis() - lastWifiConnectTime) / 60000 : 0;
+    info.wifiSignal = wifiConnected ? WiFi.RSSI() : 0;
+    info.websocketStatus = websocket.available();
+    info.websocketUptime = websocket.available() ? (millis() - lastWebsocketConnectTime) / 60000 : 0;
+    info.cpuTemp = temperatureRead();
+    return info;
```

Cppcheck (2.19.0)

[error] There is an unknown macro here somewhere. Configuration is required. If PROGMEM is a macro then please configure it.

(unknownMacro)